From cc0af81592931c40b7ebc000d8ae27559c9e3b06 Mon Sep 17 00:00:00 2001
From: Gabriele M <moto.falcon.git@gmail.com>
Date: Sat, 11 Mar 2017 20:07:59 +0100
Subject: [PATCH 118/173] Never ignore finger swipes in gallery mode

Under certain conditions finger swipes are ignored.

Steps to reproduce the bug:
 - Open left menu
 - Close menu choosing an option or tapping out of the preview
 - Tap preview of last picture in bottom left corner

Swipes in the gallry should now be ignored until a right to left
swipe is performed in the preview.

Change-Id: I3928d84360266cca368265553a037192aa5d6780
---
 src/com/android/camera/PreviewGestures.java  | 4 ----
 src/com/android/camera/ui/FilmStripView.java | 4 ----
 2 files changed, 8 deletions(-)

diff --git a/src/com/android/camera/PreviewGestures.java b/src/com/android/camera/PreviewGestures.java
index e699e56fd..bee48ad53 100644
--- a/src/com/android/camera/PreviewGestures.java
+++ b/src/com/android/camera/PreviewGestures.java
@@ -264,10 +264,6 @@ public class PreviewGestures
         return true;
     }
 
-    public boolean waitUntilNextDown() {
-        return waitUntilNextDown;
-    }
-
     private MotionEvent makeCancelEvent(MotionEvent m) {
         MotionEvent c = MotionEvent.obtain(m);
         c.setAction(MotionEvent.ACTION_CANCEL);
diff --git a/src/com/android/camera/ui/FilmStripView.java b/src/com/android/camera/ui/FilmStripView.java
index e6e00ad56..6a767862f 100644
--- a/src/com/android/camera/ui/FilmStripView.java
+++ b/src/com/android/camera/ui/FilmStripView.java
@@ -2825,8 +2825,6 @@ public class FilmStripView extends ViewGroup implements BottomControlsListener {
 
         @Override
         public boolean onScroll(float x, float y, float dx, float dy) {
-            if (mPreviewGestures != null && mPreviewGestures.waitUntilNextDown())
-                return false;
             ViewItem currItem = mViewItem[mCurrentItem];
             if (currItem == null) {
                 return false;
@@ -2892,8 +2890,6 @@ public class FilmStripView extends ViewGroup implements BottomControlsListener {
 
         @Override
         public boolean onFling(float velocityX, float velocityY) {
-            if (mPreviewGestures != null && mPreviewGestures.waitUntilNextDown())
-                return false;
             final ViewItem currItem = mViewItem[mCurrentItem];
             if (currItem == null) {
                 return false;
-- 
2.17.1

